hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2818,19 +2820,30 @@ public OzoneManagerHttpServer getHttpServer() {
             .setType(ServicePort.Type.RPC)
             .setValue(omRpcAddress.getPort())
             .build());
+
+    auditMap.put(OzoneConsts.OZONE_OM_SERVICE_HOSTNAME,
+        omRpcAddress.getHostName());
+    auditMap.put("RPC_PORT", String.valueOf(omRpcAddress.getPort()));
```

Comment:
Usually audit log should include information about the request, not the response.  Only status (success / failure) needs to be added about the response.  So for `getServiceList` request, which does not have parameters, I think we can simply log success or failure.